src/controller.coffee

```diff
@@ -1695,7 +1701,11 @@ define ['droplet-helper',
           @redrawTextInput()
 
   Editor::resizeAceElement = ->
-    @aceElement.style.width = "#{@wrapperElement.offsetWidth}px"
+    width = @wrapperElement.offsetWidth
+    if @willShowPaletteWithText and @showingPalette
```

Comment:
"New property: showingPalette - this determines whether or not the palette should appear (NOTE that it is ignored if in text mode when willShowPaletteWithText is false - this preserves the existing behavior for hosts that don't use these new options)"

This seems confusing that showingPalette doesn't necessarily reflect if we're showing the palette (i.e. if willShowPaletteWithText is false). I would expect showingPalette to always be accurate, and our other setting to determine whether the state of showingPalette changes on melt/freeze.